modules/oe_theme_content_person/oe_theme_content_person.post_update.php

```diff
@@ -82,3 +82,47 @@ function oe_theme_content_person_post_update_30001() {
     $updated_display->save();
   }
 }
+
+/**
+ * Create the 'description only' view display for the PersonJob entity type.
+ */
+function oe_theme_content_person_post_update_30002() {
+  // Create the 'Description only' view mode if it doesn't exist yet.
+  if (!EntityViewMode::load('oe_person_job.description_only')) {
+    EntityViewMode::create([
+      'id' => 'oe_person_job.description_only',
+      'targetEntityType' => 'oe_person_job',
+      'status' => TRUE,
+      'enabled' => TRUE,
+      'label' => 'Description only',
+    ])->save();
+  }
+  $storage = new FileStorage(drupal_get_path('module', 'oe_theme_content_person') . '/config/post_updates/30002_update_view_displays');
+
+  $entity_type_manager = \Drupal::entityTypeManager();
+  $config = $storage->read('core.entity_view_display.oe_person_job.oe_default.description_only');
+  // We are creating the config which means that we are also shipping
+  // it in the config/install folder so we want to make sure it gets the hash
+  // so Drupal treats it as a shipped config. This means that it gets exposed
+  // to be translated via the locale system as well.
+  $config['_core']['default_config_hash'] = Crypt::hashBase64(serialize($config));
+  /** @var \Drupal\Core\Config\Entity\ConfigEntityStorageInterface $entity_storage */
+  $entity_storage = $entity_type_manager->getStorage('entity_view_display');
+  $entity = $entity_storage->createFromStorageRecord($config);
+  $entity->save();
+
+  $display_values = $storage->read('core.entity_view_display.node.oe_person.oe_compact_teaser');
+  /** @var \Drupal\Core\Config\Entity\ConfigEntityStorageInterface $entity_storage */
+  $entity_storage = $entity_type_manager->getStorage('entity_view_display');
+  $existing_display = EntityViewDisplay::load($display_values['id']);
+  if ($existing_display) {
+    // We are creating the config which means that we are also shipping
+    // it in the config/install folder so we want to make sure it gets the hash
+    // so Drupal treats it as a shipped config. This means that it gets exposed
+    // to be translated via the locale system as well.
```

Comment:
Let's update this comment because we're not creating the config but updating it.